StratisCore.UI/src/app/shared/services/signalr-service.ts

```diff
@@ -12,7 +12,7 @@ import { Log } from '../../wallet/tokens/services/logger.service';
 
 export interface SignalRConnectionInfo {
   signalRUri: string;
-  signalRPort: string;
+  signalPort: string;
```

Comment:
Can you change this back to signalRPort?

Reply:
Will do- is this going to be changed in stratisfullnode? Otherwise, it causes the con.signalRPort on line 62 to be undefined